app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +261,67 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = (
+    <div>
+      {menuDropDown(data).map(
+        ({
+          name,
+          notificationCount,
+          route,
+          target,
+          externalLink,
+          hasBottomDivider,
+        }) => {
+          const hasNotification =
+            notificationCount !== undefined && notificationCount > -1;
```

Comment:
```suggestion
            notificationCount !== undefined && notificationCount > 0;
```
I think what you had before was correct, as now the notification dot is showing all the time with "0 unseen"

Reply:
Yes i was doing some test but i now removed it and make sure it'll be pushed as "0" :)